v1/app/views/users_views.py

```diff
@@ -0,0 +1,38 @@
+from flask import Blueprint
+from flask_restful import reqparse
+
+# local imports
+from app.controllers.users_controllers import Users
+
+don_user = Blueprint('users', __name__, url_prefix="/api/v1")
+
+user_insatnce = Users()
```

Comment:
Isn't this a typo?

Reply:
what part exactly?

Reply:
@Ruiru11 Do you mean `user_instance` or `user_insatnce`? `cc: @danuluma`

Reply:
it should be instance thanks